alacritty/src/main.rs

```diff
@@ -79,18 +82,23 @@ fn main() {
 
     #[cfg(unix)]
     let result = match options.subcommands {
-        Some(Subcommands::Msg(options)) => msg(options),
+        Some(Subcommands::Msg(options)) => msg(options).map(|_| EXIT_SUCCESS),
         None => alacritty(options),
     };
 
     #[cfg(not(unix))]
     let result = alacritty(options);
 
     // Handle command failure.
-    if let Err(err) = result {
-        eprintln!("Error: {}", err);
-        process::exit(1);
-    }
+    let exit_code = match result {
+        Ok(code) => code,
+        Err(err) => {
+            eprintln!("Error: {}", err);
+            EXIT_FAILURE
+        },
+    };
```

Comment:
Why bother with exit codes and all that stuff when `Err(err)` is mapped to an exit code anyway? Just so we don't print the error?

Reply:
The idea was to propagate the error code from the event loop somehow, since now `run_return` will return non zero code in case of failures.

I basically wanted to indicate an error with non zero exit status nothing more. If you have an idea on how to handle that more reliably let me know.

Reply:
Why not return the actual error? That seems to be the sensible choice to me.